requirements-freeze.txt

```diff
@@ -74,5 +74,5 @@ xlrd==1.2.0
 xlutils==2.0.0
 xlwt==1.3.0
 zipp==0.6.0
-torchvision==0.6.0+cpu
-torch==1.5.0+cpu
+torchvision==0.6.0+cpu -f https://download.pytorch.org/whl/torch_stable.html
```

Comment:
please do not touch the requirements-freeze (this file should not be present in your dev version-- it is only for the stable version)

Reply:
OK I'll find a way to delete my modification! I modified it because travis was using it for the installation.

Reply:
> OK I'll find a way to delete my modification! I modified it because travis was using it for the installation.

it's using it because you have it  😉 --> see [here](https://github.com/neuropoly/spinalcordtoolbox/blob/master/install_sct#L544)

Reply:
Yeah I just figured it out !